src/android/java/org/alohalytics/HttpTransport.java

```diff
@@ -150,14 +158,13 @@ public static Params run(final Params p) throws IOException, NullPointerExceptio
     public String url = null;
     // Can be different from url in case of redirects.
     public String receivedUrl = null;
-    // SHOULD be specified for any POST request (any request where we send data to the server).
+    public String httpMethod = null;
+    // SHOULD be specified for any POST-like request (any request where we send data to the server).
     // On return, contains received Content-Type or null.
     public String contentType = null;
-    // Can be specified for any POST request (any request where we send data to the server).
+    // Can be specified for any POST-like request (any request where we send data to the server).
```

Comment:
'POST-like' sounds a bit confusing.
Maybe something like 'Can be specified for any request whose method allows non-empty body.'